src/components/Todo/Todo.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import cn from "classnames";
+import { useSelector, useDispatch } from "react-redux";
+import { getTodoId } from "../../store";
+import { updateUserID, updateTodoId, clear } from "../../store/actions";
+
+interface Todo {
+  id: number;
+  title: string;
+  userId: number;
+  completed: boolean;
+}
+
+export const Todo: React.FC<{ todo: Todo }> = ({ todo }) => {
+  const { id, title, userId, completed } = todo;
+  const activeTodo = useSelector(getTodoId);
+
+  const dispatch = useDispatch();
+
+  return (
+    <li
+      className={`TodoList__item ${
+        completed ? "TodoList__item--checked" : "TodoList__item--unchecked"
+      }`}
```

Comment:
consider to use `classnames` lib for conditional styles

Reply:
done